javascript/src/main/pages/TutorAssignment/TutorAssignments.js

```diff
@@ -4,31 +4,87 @@ import { Button } from "react-bootstrap";
 import { fetchWithToken } from "main/utils/fetch";
 import { useAuth0 } from "@auth0/auth0-react";
 import Loading from "main/components/Loading/Loading";
-import TutorAssignmentTable from "main/components/TutorAssignment/TutorAssignmentTable"
+import TutorAssignmentTable from "main/components/TutorAssignment/TutorAssignmentTable";
+import { TutorAssignmentCSVButton } from "main/components/TutorAssignment/TutorAssignmentCSVButton";
+import { useToasts } from "react-toast-notifications";
+
+import { uploadTutorAssignmentCSV } from "main/services/TutorAssignment/TutorAssignmentService";
 import { buildDeleteTutorAssignment } from "main/services/TutorAssignment/TutorAssignmentService";
-import {useHistory} from "react-router-dom";
+
+import { useHistory } from "react-router-dom";
 import { CSVLink } from "react-csv";
 
 const TutorAssignment = () => {
   const history = useHistory();
   const { user, getAccessTokenSilently: getToken } = useAuth0();
   const { email } = user;
+  const { addToast } = useToasts();
 
-  const { data: roleInfo } = useSWR(
-      ["/api/myRole", getToken],
-      fetchWithToken
-  );
+  const { data: roleInfo } = useSWR(["/api/myRole", getToken], fetchWithToken);
 
   const { data: instructorCourseList } = useSWR(
     [`/api/member/courses/forInstructor/${email}`, getToken],
     fetchWithToken
   );
 
-  const isInstructor = roleInfo && roleInfo.role && instructorCourseList && (instructorCourseList.length > 0 || roleInfo.role.toLowerCase() === "admin");
-  
-  const { data: tutorAssignmentList, error, mutate: mutateCourses} = useSWR(
-    ["/api/member/tutorAssignments", getToken],
-    fetchWithToken
+  const isInstructor =
+    roleInfo &&
+    roleInfo.role &&
+    instructorCourseList &&
+    (instructorCourseList.length > 0 ||
+      roleInfo.role.toLowerCase() === "admin");
+
+  const {
+    data: tutorAssignmentList,
+    error,
+    mutate: mutateTutorAssignments, mutateCourses,
```

Comment:
We can remove the mutateCourse reference here.

Reply:
done